examples/converters.py

```diff
@@ -71,17 +72,17 @@ async def convert(self, ctx: commands.Context, argument: str):
             return channel
 
         # If the value could not be converted we can raise an error
-        # So our error handlers can deal with it in one place.
+        # so our error handlers can deal with it in one place.
         # The error has to be CommandError derived, so BadArgument works fine here.
         raise commands.BadArgument(f'No Member or TextChannel could be converted from "{argument}"')
 
 
 @bot.command()
 async def notify(ctx: commands.Context, target: ChannelOrMemberConverter):
-    # This command signature utilises the custom converter written above
```

Comment:
This doesn’t need to be changed. I know people from the UK which call utilizes, utilises. It’s like color and colour.

Reply:
Reverted

Reply:
Can vouch here, the UK uses `s` for things like utilise, specialise, etc